### PR TITLE
Add documentation about the tests.

### PR DESCRIPTION
**Pull request by dragotin**
I tried to describe the tests to make it easier to understand what the tests actually do and how they are relevant.

Thanks for good review :-)

### CODE REVIEW

File: docs/tests.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+The files are created, uploaded, downloaded and afterwards deleted in this test. The current implementation tests with in total 15GB data per User.
+
+This simulates a "normal" upload/download scenario with a common file set.
+
+### Deep Propfind
+
+Script: `/root/cdperf/tests/k6/test-issue-github-ocis-1018-propfind-deep.js`
+
+This test creates a nested directory structure that is fife levels deep and distributes 1000 files within that tree.
```
Comment (fschade) [inline docs/tests.md:20]: Five

File: docs/tests.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+### Flat Propfind
+
+Script: `/root/cdperf/tests/k6/test-issue-github-ocis-1018-propfind-flat.js`
+
+This test crates many (1000 a 1kB) files in the root of the cloud.
+
+The files are uploaded and afterwards, a PROPFIND to the root directory is done.
+
+This test gives an indication on the PROPFIND speed of a large root and also measures the pure upload speed of small files.
```
Comment (fschade) [inline docs/tests.md:34]: … for a larger structure of nested folders …

File: docs/tests.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+### Upload Big Files
+
+Script: `/root/cdperf/tests/k6/test-issue-github-ocis-1018-upload-download-delete-many-large.js`
+
+This test creates in sum 5 GB of files larger than 5 MB and up to 1GB.
```
Comment (fschade) [inline docs/tests.md:40]: In total this test creates 5gb of files which distributes from 5mb up to 1gb per file

File: docs/tests.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+The files are uploaded, downloaded again and deleted afterwards.
```
Comment (fschade) [inline docs/tests.md:42]: The files are uploaded, downloaded and deleted afterwards.

File: docs/tests.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+This gives indication about the transmission speed of big files.
+
+### Upload Small Files
+
+Script: `/root/cdperf/tests/k6/test-issue-github-ocis-1018-upload-download-delete-many-small.js`
+
+This test creates files between 500 kB and 25 MB, in sum 1.7 GB.
```
Comment (fschade) [inline docs/tests.md:50]: with a total size of …

File: docs/tests.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+The script creates 50 folders in the root with in sum 10 MB content files.
+
+The folders and files are uploaded, renamed and after the rename, a PROPFIND is done.
+
+This reflects the behaviour of renames of folders.
+
+### Share with User
+
+This script creates a folder and fills it with test files.
```
Comment (fschade) [inline docs/tests.md:76]: This script creates a folder and provisions the folder with some files right after

File: docs/tests.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+After upload of the test folder, it is shared with a different user. With the share receiver, the files are downloaded. Finally the share is removed and the user and files are deleted.
```
Comment (fschade) [inline docs/tests.md:78]: … right after the test folder is uploaded, it will be shared to ….

**Review by fschade — APPROVED** [review]
Some small findings, else good Tongi. Thanks for taking care! 😘 

🚀 LGTM

**Comment by ScharfViktor** [comment]
Oops, looks like I didn't make it

**Pull request merged**